Example/Tests/Security/CyclicCATests.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  OngoingAccessTests.swift
+//  DigiMeSDKExample_Tests
+//
+//  Created on 02/12/2019.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+import CryptorRSA
+import Foundation
+import SwiftJWT
+import XCTest
+
+class OngoingAccessTests: XCTestCase {
+    struct Keys {
+        static let token = "jwtToken"
+        static let title = "title"
+        static let valid = "valid"
+    }
+    
+    struct TestClaimsVerification: Claims {
+        let test: String
+    }
+    
+    struct TestClaimsDecoding: Claims {
+        let preauthorization_code: String
+    }
+    
+    var header: Header!
+    var publicKey: Data!
+    var privateKey: Data!
+    var testingAssets: [AnyHashable: Any]!
+    
+    override func setUp() {
+        guard
+            let assetsPath = Bundle(for: type(of: self)).path(forResource: "CyclicCATestingAssets", ofType: "plist"),
+            let assets = NSDictionary(contentsOfFile: assetsPath) as? [AnyHashable: Any],
+            let publicKeyBase64 = assets["publicKey"] as? String,
+            let privateKeyBase64 = assets["privateKey"] as? String,
+            let publicKeyData = convertKeyString(publicKeyBase64),
+            let privateKeyData = convertKeyString(privateKeyBase64) else {
+                XCTFail("Failed to load required testing assets.")
+                return
+        }
+        
+        publicKey = publicKeyData
+        privateKey = privateKeyData
+        header = Header(typ: "JWT",
+                        jku: "https://api.development.devdigi.me/v1/jwks/oauth",
+                        jwk: nil,
+                        kid: "https://digime-alpha-key-vault.vault.azure.net/keys/jfs-signing/a46b49e1fd66470d9ef7d444b5429243",
```

Comment:
Should we really be connecting to live services for testing? Shouldn't this be stubbed?